### ISSUE TITLE
Bug: `node:test` is not found

### ISSUE THREAD

**Issue by gurgunday** [eslint-community/eslint-plugin-n#232]
### Environment

Node version: 20.12.1
ESLint version: v8
eslint-plugin-n version: 17.0.0

### What rule do you want to report?

n/no-missing-import

### Link to Minimal Reproducible Example

X

### What did you expect to happen?

The following should work:

```js
import test from "node:test";
```

### Participation

- [x] I am willing to submit a pull request for this issue.
